frontend/pages/DetailedViewPage.js

```diff
@@ -89,15 +120,35 @@ export default class DetailedViewPage extends Component {
 const styles = StyleSheet.create({
```

Comment:
I get that you need these large styleSheets, but is not possible to have these in a separate file? Perhaps you could have a directory of style sheets which you could choose/reuse from? That way styling would be more consistent across the app?

Reply:
We can fix this. We have been planning on standardizing styling across the app; we can do this with your suggestion or keep the styleSheets and just standardize all of them

Reply:
From a style perspective we decided to keep the stylesheet associated with a page in the same file so that developers don't need to modify multiple files to modify a single page. I do concede that having some global stylesheets would help unify style and I have made an issue (#127) to address this.